docs/admin/node-allocatable.md

```diff
@@ -130,21 +197,34 @@ Here is an example to illustrate Node Allocatable computation:
 * `--eviction-hard` is set to `memory.available<500Mi`
 
 Under this scenario, `Allocatable` will be `14.5 CPUs` & `28.5Gi` of memory.
-Scheduler ensures that the total `requests` across all pods on this node does not exceed `28.5Gi`.
-Kubelet evicts pods whenever the overall memory usage exceeds across pods exceed `28.5Gi`.
-If all processes on the node consume as much CPU as they can, pods together cannot consume more than `14.5 CPUs`.
+Scheduler ensures that the total `requests` across all pods on this node does
+not exceed `28.5Gi`. Kubelet evicts pods whenever the overall memory usage
+exceeds across pods exceed `28.5Gi`. If all processes on the node consume as
+much CPU as they can, pods together cannot consume more than `14.5 CPUs`.
 
-If `kube-reserved` and/or `system-reserved` is not enforced and system daemons exceed their reservation, `kubelet` evicts pods whenever the overall node memory usage is higher than `31.5Gi`.
+If `kube-reserved` and/or `system-reserved` is not enforced and system daemons
+exceed their reservation, `kubelet` evicts pods whenever the overall node memory
+usage is higher than `31.5Gi`.
 
 ## Feature Availability
 
-Since `v1.2`, it has been possible to **optionally** specify `kube-reserved` and `system-reserved` reservations.
-The scheduler switched to using `Allocatable` instead of `Capacity` when available in the same release.
-
-Since `v1.6`, `eviction-thresholds` are being considered by computing `Allocatable`.
-To revert to the old behavior set `--experimental-allocatable-ignore-eviction` kubelet flag to `true`.
-
-Since `v1.6`, `kubelet` enforces `Allocatable` on pods using control groups.
-To revert to the old behavior unset `--enforce-node-allocatable` kubelet flag.
-Note that unless `--kube-reserved`, or `--system-reserved` or `--eviction-hard` flags have non-default values, `Allocatable` enforcement does not affect existing deployments.
+Since `v1.2`, it has been possible to **optionally** specify `kube-reserved` and
```

Comment:
"Since v1.2" -> "As of Kubernetes version 1.2"

Repeat for all version numbers below.